src/miner.cpp

```diff
@@ -932,7 +932,7 @@ void BitcoinMiner(std::shared_ptr<CReserveScript> coinbaseScript, bool fProofOfS
                 auto it = mapStakeHashCounter.find(nHeight);
                 if (it != mapStakeHashCounter.end() && it->second != nStakeHashesLast) {
                     nStakeHashesLast = it->second;
-                    LogPrint(BCLog::BLOCKCREATION, "%s: Tried %d stake hashes for block %d last=%d\n", __func__, nStakeHashesLast, nHeight+1, mapHashedBlocks.at(hashBestBlock));
+                    LogPrint(BCLog::MINING, "%s: Tried %d stake hashes for block %d last=%d\n", __func__, nStakeHashesLast, nHeight+1, mapHashedBlocks.at(hashBestBlock));
```

Comment:
This one is specific to Proof of Stake rather than Proof of Work, and relevant to watching staking rather than mining.  So this one needs to stay at BlockCreation (and perhaps will be split off from Blockcreation in a separate PR to have staking separated).  But for now, it should stay in the BLOCKCREATION group.

Reply:
Done.